gillespy2/solvers/numpy/tau_hybrid_solver.py

```diff
@@ -1137,30 +1133,25 @@ def __run(self, curr_state, curr_time, timeline, trajectory_base, initial_state,
         t0_delayed_events, species_modified_by_events = self.__check_t0_events(initial_state)
 
         # Create deterministic tracking data structures
-        det_spec = {species: True for (species, value) in self.model.listOfSpecies.items() if value.mode == 'dynamic'}
+        #det_spec = {species: True for (species, value) in self.model.listOfSpecies.items() if value.mode == 'dynamic'}
+        det_spec = {species: False for species in self.model.listOfSpecies.keys()}
```

Comment:
We could still do this as a dictionary comprehension:
`det_spec = {species: True for (species, value) in self.model.listOfSpecies.items() if value.mode != 'discrete'}`

Not sure if there is much performance to be gained from that, though.